### DISCUSSION TITLE
Hero 5 - lab firmware

### DISCUSSION THREAD

**Discussion by char1976** [gopro/labs#318] [Q&A]
I am looking to download the GoPro labs firmware onto a few Hero 5's. I know it is only listed as working on the Hero 5 sessions, but I would like to know if there is a download for the Hero 5 that may have been archived that will enable QR codes (for time delays) or if it just doesn't exist for these GoPros? Thanks

**Comment by dnewman-gpsw** [comment]
It was never released for either HERO5 or HERO6.  Labs came out first on HERO8, only HERO7 and HERO5S where retro releases (in 2020).  Sorry there won't be any more now super-retro releases.

**Reply by char1976** [reply]
Thank you very much for confirming
